frontend/src/views/main/admin/settings/SettingsGeneral.vue

```diff
@@ -79,6 +83,11 @@
           <ApiTokenCard />
         </v-col>
       </v-row>
+      <v-row>
+        <v-col>
+          <ClientInstructionCard />
```

Comment:
Got it!
I thought that Settings are always accessible to everyone, and not the case for the "admin" page (which is indeed `UserProfile` page of admin user, instead of an admin page).